cryodrgn/ctf.py

```diff
@@ -7,7 +8,17 @@
 log = utils.log
 
 
-def compute_ctf(freqs, dfu, dfv, dfang, volt, cs, w, phase_shift=0, bfactor=None):
+def compute_ctf(
+    freqs,
+    dfu,
+    dfv,
+    dfang,
+    volt,
+    cs,
+    w,
+    phase_shift: Union[torch.Tensor, float] = 0,
```

Comment:
This is the kind of thing I'd like to get rid of, but it's tricky because it requires understanding the semantics of the code.

If you end up doing this a lot, perhaps typedef this somewhere.

Btw why are you adding a type to this argument and nothing else?

Reply:
1. You mean you'd like to get rid of these because it's too verbose (in which case, I agree we should put this in `cryodrgn.types` and give it a name, or like to refactor the code to avoid having multiple types as inputs? If the latter, then I think we better do it as separate tiny PRs on a case-by-case basis later.

2. I think this parameter must have been touched by some code path that I was trying to `pyright` and fix. I'm making no claims yet on being complete annotation-complete in the files that are in `include` (perhaps I should be?), but simply trying to get `pyright` to not fail on any files that *are* included.